app/views/admin/administrators/administrators/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Administrators</h1>
+    <%= govuk_link_to "Create a new administrator", new_admin_administrators_administrator_path, button: true %>
+    <br/>
```

Comment:
Is this just for spacing? If so, let's use a margin-bottom (or margin-top) class from govuk design system